101-lustre-client-server-create/README.md

```diff
@@ -0,0 +1,18 @@
+﻿# Lustre client and server node VMs deployment
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-lustre-client-server-create/azuredeploy.json" target="_blank">
+<img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-lustre-client-server-create/azuredeploy.json" target="_blank">
+<img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+
```

Comment:
I'm not seeing this in the latest round of updates. Can you link to where this info is?